packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -385,7 +408,8 @@ export class MaterialPanel extends ConnectedLitElement {
 
     const id = this.selectedMaterialId;
     const textureId = this.selectedOcclusionTextureId;
-    dispatchOcclusionTexture({id, textureId});
+    reduxStore.dispatch(dispatchOcclusionTexture(
+        getEditsMaterials(reduxStore.getState()), {id, textureId}));
```

Comment:
I haven't looked through this enough to know how universal this is, but it seems like `reduxStore.getState()` is always the argument. Why can't it just be implied? Not a critique, I'm just trying to understand Redux better.

Reply:
this has to do with circular dependencies. if we don't pass it, we'd need to either (a) create a separate selectors file (instead of having them be alongside the reducer) or (b) have a circular dependency between: 
component -> store -> reducer -> store

Reply:
In general, your components have access to the state from import/wrapper, but your reducers are passed the state from functions

Reply:
k, I'll trust you.